drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-businesscentral-client/src/main/java/org/drools/workbench/screens/scenariosimulation/businesscentral/client/editor/ScenarioSimulationEditorBusinessCentralWrapper.java

```diff
@@ -276,9 +278,7 @@ protected Command getBeforeSaveAndRenameCommand() {
     }
 
     protected void synchronizeColumnsDimension() {
-        final ScenarioGridModel scenarioGridModel = scenarioSimulationEditorPresenter.getView().getScenarioGridLayer()
-                .getScenarioGrid().getModel();
-        scenarioGridModel.synchronizeFactMappingsWidths();
+        scenarioSimulationEditorPresenter.getFocusedContext().getScenarioGridPanel().synchronizeFactMappingsWidths();
```

Comment:
@danielezonca Just because I didn't consider it as a part of this PR (managing persistent widths for background). I propose to handle it in the PR which will introduce the DataModel to handle background data

Reply:
Ok can you please create a ticket for that?